craft_parts/overlays/chroot.py

```diff
@@ -32,7 +32,12 @@
 logger = logging.getLogger(__name__)
 
 
-def chroot(path: Path, target: Callable, *args: Any, **kwargs: Any) -> Any:
+def chroot(
+    path: Path,
+    target: Callable,  # type: ignore[type-arg]
+    *args: Any,  # noqa: ANN401
+    **kwargs: Any,  # noqa: ANN401
```

Comment:
We can add proper typing for `args` and `kwargs` instead of disabling the warning.